jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/WebAppInventoryCollector.java

```diff
@@ -28,10 +29,9 @@ public List<WebAppVH> fetchWebAppDetails(SubscriptionVH subscription) {
         Azure azure = azureCredentialProvider.getClient(subscription.getTenant(), subscription.getSubscriptionId());
 
         PagedList<WebApp> webApps = azure.webApps().list();
-
+        WebAppVH webAppVH = new WebAppVH();
```

Comment:
Why move it here?

Reply:
Need to get region and subscription in ->  ErrorManageUtil.uploadError(webAppVH.getSubscription(),webAppVH.getRegion(),"webapp",exception.getMessage());